src/server.c

```diff
@@ -220,7 +220,15 @@ int dqlite_node_set_bind_address(dqlite_node *t, const char *address)
 			/* Auto bind */
 			len = 0;
 		} else {
+#if defined(__linux__)
+			// Linux abstract socket requires \0 in sun_path[0]
 			strcpy(addr_un.sun_path + 1, address + 1);
```

Comment:
Folks, maybe to use strncpy/strcat here? For now I left it as is, but maybe we need little more fortification?